stories/wigi.md

```diff
@@ -10,9 +10,20 @@
 .. template: mainpage.tmpl
 -->
 
-GENDER GAP THROUGH TIME AND SPACE : A JOURNEY THROUGH WIKIPEDIA BIOGRAPHIES
-===========================================================================
+<strong>Wi</strong>kipedia <strong>G</strong>ender Indicators (<strong>WIGI</strong>)
+-------------------------------------------------------------------------------------
 
-It is an unfortunate but unavoidable fact that encyclopedias always have had a gender bias. One of the dimensions of this bias is the contributors’ gender distribution (Thomas 1992; Reagle and Rhue 2011). Just as there were very few women authors among contributors to traditional, printed encyclopedias, recent surveys indicate that women constitute only around 13%-16% of Wikipedia contributors, or Wikipedians (Glott, Schmidt, & Ghosh 2010; Hill and Shaw 2013). 
+![36th_g8_summit_member_cropped_36th_g8_summit_member_20100625_1080](https://cloud.githubusercontent.com/assets/1494283/10463045/bfbe87f4-71a7-11e5-9851-59778a9f9ee0.jpg)
```

Comment:
Why do we have this photograph of bureaucrats here? I guess a better alternative will be the infographic Frances made.